2024-12-11 10:00:26,402 - ERROR - URL Error http://204.157.251.250:8000/play/a068/index.m3u8: HTTPConnectionPool(host='204.157.251.250', port=8000): Max retries exceeded with url: /play/a068/index.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9f1ad6bd70>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-11 10:00:28,200 - ERROR - URL Error https://cnn-cnninternational-1-eu.rakuten.wurl.tv/playlist.m3u8: HTTPSConnectionPool(host='cnn-cnninternational-1-eu.rakuten.wurl.tv', port=443): Max retries exceeded with url: /playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc03b0>: Failed to resolve 'cnn-cnninternational-1-eu.rakuten.wurl.tv' ([Errno -2] Name or service not known)"))
2024-12-11 10:00:46,825 - ERROR - URL Error http://cord-cutter.net:8080/live/j3McKd/673709/57619.m3u8: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-11 10:01:22,838 - ERROR - URL Error http://cloud.logicwebs.in:1935/live/malabarnews/playlist.m3u8: HTTPConnectionPool(host='cloud.logicwebs.in', port=1935): Max retries exceeded with url: /live/malabarnews/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9f1adc0230>, 'Connection to cloud.logicwebs.in timed out. (connect timeout=15)'))
2024-12-11 10:01:37,854 - ERROR - URL Error http://103.199.161.254/Content/manoramanews/Live/Channel(ManoramaNews)/index.m3u8: HTTPConnectionPool(host='103.199.161.254', port=80): Max retries exceeded with url: /Content/manoramanews/Live/Channel(ManoramaNews)/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9f1adc2990>, 'Connection to 103.199.161.254 timed out. (connect timeout=15)'))
2024-12-11 10:01:52,871 - ERROR - URL Error http://103.199.161.254/Content/mathrubhuminews/Live/Channel(Mathrubhuminews)/index.m3u8: HTTPConnectionPool(host='103.199.161.254', port=80): Max retries exceeded with url: /Content/mathrubhuminews/Live/Channel(Mathrubhuminews)/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9f1adc3260>, 'Connection to 103.199.161.254 timed out. (connect timeout=15)'))
2024-12-11 10:01:53,955 - ERROR - URL Error http://edge.metrotvnews.com:1935/live-edge/smil:metro.smil/playlist.m3u8: HTTPConnectionPool(host='edge.metrotvnews.com', port=1935): Max retries exceeded with url: /live-edge/smil:metro.smil/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9f1adc3f80>: Failed to resolve 'edge.metrotvnews.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:01:55,138 - ERROR - URL Error https://rakuten-euronews-8-pt.samsung.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='rakuten-euronews-8-pt.samsung.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc3920>: Failed to resolve 'rakuten-euronews-8-pt.samsung.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:01:58,527 - ERROR - URL Error https://d2xqcdx4c8vu9s.cloudfront.net/out/v1/15b8393e9a9240b8bedd4c79ccefb301/index.m3u8: HTTPSConnectionPool(host='d2xqcdx4c8vu9s.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/15b8393e9a9240b8bedd4c79ccefb301/index.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc1eb0>: Failed to resolve 'd2xqcdx4c8vu9s.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-11 10:02:03,614 - ERROR - URL Error http://cdn2.cyclingnews.live/hls/P2.m3u8: HTTPConnectionPool(host='cdn2.cyclingnews.live', port=80): Max retries exceeded with url: /hls/P2.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9f1adc11f0>: Failed to resolve 'cdn2.cyclingnews.live' ([Errno -2] Name or service not known)"))
2024-12-11 10:02:20,140 - ERROR - URL Error https://bloomberg-bloomberg-1-be.samsung.wurl.tv/playlist.m3u8: HTTPSConnectionPool(host='bloomberg-bloomberg-1-be.samsung.wurl.tv', port=443): Max retries exceeded with url: /playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc2090>: Failed to resolve 'bloomberg-bloomberg-1-be.samsung.wurl.tv' ([Errno -2] Name or service not known)"))
2024-12-11 10:02:26,124 - ERROR - URL Error https://jobtvvideo.space/hls/stream/index.m3u8: HTTPSConnectionPool(host='jobtvvideo.space', port=443): Max retries exceeded with url: /hls/stream/index.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1addc170>: Failed to resolve 'jobtvvideo.space' ([Errno -2] Name or service not known)"))
2024-12-11 10:02:30,552 - ERROR - URL Error https://d8gy12azhr71i.cloudfront.net/out/v1/45be109c5b4f44319e882da947377364/index_3.m3u8: HTTPSConnectionPool(host='d8gy12azhr71i.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/45be109c5b4f44319e882da947377364/index_3.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc1400>: Failed to resolve 'd8gy12azhr71i.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-11 10:02:40,873 - ERROR - URL Error https://securecontributions.sechls01.visionip.tv/live/securecontributions-securecontributions-92_news-hsslive-25f-16x9-SD/chunklist.m3u8: HTTPSConnectionPool(host='securecontributions.sechls01.visionip.tv', port=443): Max retries exceeded with url: /live/securecontributions-securecontributions-92_news-hsslive-25f-16x9-SD/chunklist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc3b30>: Failed to resolve 'securecontributions.sechls01.visionip.tv' ([Errno -3] Temporary failure in name resolution)"))
2024-12-11 10:02:56,707 - ERROR - URL Error https://evpp.mm.uol.com.br/geob_band/agromais/playlist.m3u8: HTTPSConnectionPool(host='evpp.mm.uol.com.br', port=443): Max retries exceeded with url: /geob_band/agromais/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9f1adc2ea0>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-11 10:03:00,537 - ERROR - URL Error https://streamcdnb10-859c1818ed614cc5b0047439470927b0.msvdn.net/live/S76890577/tDoFkZD3T1Lw/chunklist_b1728000.m3u8: HTTPSConnectionPool(host='streamcdnb10-859c1818ed614cc5b0047439470927b0.msvdn.net', port=443): Max retries exceeded with url: /live/S76890577/tDoFkZD3T1Lw/chunklist_b1728000.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc3290>: Failed to resolve 'streamcdnb10-859c1818ed614cc5b0047439470927b0.msvdn.net' ([Errno -2] Name or service not known)"))
2024-12-11 10:03:00,562 - ERROR - URL Error https://d8gy12azhr71i.cloudfront.net/out/v1/45be109c5b4f44319e882da947377364/index_5.m3u8: HTTPSConnectionPool(host='d8gy12azhr71i.cloudfront.net', port=443): Max retries exceeded with url: /out/v1/45be109c5b4f44319e882da947377364/index_5.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1addcd10>: Failed to resolve 'd8gy12azhr71i.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-11 10:03:14,558 - ERROR - URL Error http://cord-cutter.net:8080/k4Svp2/645504/1818#checkedby:alliptvlinks.com: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-11 10:03:17,967 - ERROR - URL Error https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/news24local.stream/playlist.m3u8: HTTPSConnectionPool(host='cdn.appv.jagobd.com', port=444): Max retries exceeded with url: /c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/news24local.stream/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9f1adc33b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-11 10:03:25,223 - ERROR - URL Error https://okkotv-live.cdnvideo.ru/channel/Russia24.m3u8: HTTPSConnectionPool(host='okkotv-live.cdnvideo.ru', port=443): Max retries exceeded with url: /channel/Russia24.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc1340>: Failed to resolve 'okkotv-live.cdnvideo.ru' ([Errno -2] Name or service not known)"))
2024-12-11 10:03:40,462 - ERROR - URL Error https://gtrkchita.ru:8081/hls/r24-chita_576p.m3u8: HTTPSConnectionPool(host='gtrkchita.ru', port=8081): Max retries exceeded with url: /hls/r24-chita_576p.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9f1adc2780>, 'Connection to gtrkchita.ru timed out. (connect timeout=15)'))
2024-12-11 10:03:56,028 - ERROR - URL Error http://ott-cdn.ucom.am/s11/index.m3u8: HTTPConnectionPool(host='ott-cdn.ucom.am', port=80): Max retries exceeded with url: /s11/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9f1ad6a9c0>, 'Connection to ott-cdn.ucom.am timed out. (connect timeout=15)'))
2024-12-11 10:03:58,482 - ERROR - URL Error https://bloomberg-bloombergtv-1-it.samsung.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='bloomberg-bloombergtv-1-it.samsung.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1addc6b0>: Failed to resolve 'bloomberg-bloombergtv-1-it.samsung.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:03:58,569 - ERROR - URL Error https://bloomberg-bloomberg-1-be.samsung.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='bloomberg-bloomberg-1-be.samsung.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1addc410>: Failed to resolve 'bloomberg-bloomberg-1-be.samsung.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:04:16,539 - ERROR - URL Error https://ndtv.live-s.cdn.bitgravity.com/cdn-live/_definst_/ndtv/live/ndtv_profit.smil/playlist.m3u8: HTTPSConnectionPool(host='ndtv.live-s.cdn.bitgravity.com', port=443): Max retries exceeded with url: /cdn-live/_definst_/ndtv/live/ndtv_profit.smil/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9f1adc0b90>, 'Connection to ndtv.live-s.cdn.bitgravity.com timed out. (connect timeout=15)'))
2024-12-11 10:04:23,315 - ERROR - URL Error https://movie.mcas.jp/mcas/smil:wn1.smil/master.m3u8: HTTPSConnectionPool(host='movie.mcas.jp', port=443): Max retries exceeded with url: /mcas/smil:wn1.smil/master.m3u8 (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'movie.mcas.jp'. (_ssl.c:1000)")))
2024-12-11 10:04:27,100 - ERROR - URL Error https://reuters-reuters-1-eu.rakuten.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='reuters-reuters-1-eu.rakuten.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc0ec0>: Failed to resolve 'reuters-reuters-1-eu.rakuten.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:04:27,119 - ERROR - URL Error https://reuters-reutersnow-1.plex.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='reuters-reutersnow-1.plex.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adc0c20>: Failed to resolve 'reuters-reutersnow-1.plex.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:04:27,134 - ERROR - URL Error https://reuters-reutersnow-1.plex.wurl.com/manifest/c0c0829237382f291cc46a0368586aa7.m3u8: HTTPSConnectionPool(host='reuters-reutersnow-1.plex.wurl.com', port=443): Max retries exceeded with url: /manifest/c0c0829237382f291cc46a0368586aa7.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1addcdd0>: Failed to resolve 'reuters-reutersnow-1.plex.wurl.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:04:55,203 - ERROR - URL Error https://cdnswitch-mediahub.overon.es/stream/stream.m3u8?key=R6w5wbcxW3dLF9aNOGEsHuoNhqfSJoiRWNDWLLfNslI&pwd=QheEdHkaeHFGDHJgIwzUDoqcVjjFMef8TMGuGvkMets&stream=y3-IxQxNk4XjlfJDtBjIh7tfO628d9gbkd010uKioms&output=ginkIjHWeyffMLRZNMVovht3RnH_feedjshkMm7sr14&channel=D1tXjvcs_5NeEEMxP6mrmSggh3Uyu3kE5jyqGlgCZ4A&protocol=4CMm-Q0epGZ0SHQGeELoqqA2gfzFVCFwoiACYszyPM4&type=3pIgQ1TALsxZg7Zf0kZLkA4InWPfKONROAImuoufxKg: HTTPSConnectionPool(host='cdnswitch-mediahub.overon.es', port=443): Max retries exceeded with url: /stream/stream.m3u8?key=R6w5wbcxW3dLF9aNOGEsHuoNhqfSJoiRWNDWLLfNslI&pwd=QheEdHkaeHFGDHJgIwzUDoqcVjjFMef8TMGuGvkMets&stream=y3-IxQxNk4XjlfJDtBjIh7tfO628d9gbkd010uKioms&output=ginkIjHWeyffMLRZNMVovht3RnH_feedjshkMm7sr14&channel=D1tXjvcs_5NeEEMxP6mrmSggh3Uyu3kE5jyqGlgCZ4A&protocol=4CMm-Q0epGZ0SHQGeELoqqA2gfzFVCFwoiACYszyPM4&type=3pIgQ1TALsxZg7Zf0kZLkA4InWPfKONROAImuoufxKg (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9f1adc3bc0>, 'Connection to cdnswitch-mediahub.overon.es timed out. (connect timeout=15)'))
2024-12-11 10:05:10,339 - ERROR - URL Error https://cdnswitch-mediahub.overon.es/stream/stream.m3u8?key=R6w5wbcxW3dLF9aNOGEsHuoNhqfSJoiRWNDWLLfNslI&pwd=QheEdHkaeHFGDHJgIwzUDoqcVjjFMef8TMGuGvkMets&stream=o0RCNMRf-BPdBiuDSId7vbxANj4YFVpbXkxNkFKm35s&output=z3mwepuDBMOnja63VsgAcCGwmmOPD-I5yFYOiyCvBcc&channel=FBDoImE2NP77XpNbdycjnwCkI0YjxLeqTKr8QGV9bAI&protocol=4CMm-Q0epGZ0SHQGeELoqqA2gfzFVCFwoiACYszyPM4&type=3pIgQ1TALsxZg7Zf0kZLkA4InWPfKONROAImuoufxKg: HTTPSConnectionPool(host='cdnswitch-mediahub.overon.es', port=443): Max retries exceeded with url: /stream/stream.m3u8?key=R6w5wbcxW3dLF9aNOGEsHuoNhqfSJoiRWNDWLLfNslI&pwd=QheEdHkaeHFGDHJgIwzUDoqcVjjFMef8TMGuGvkMets&stream=o0RCNMRf-BPdBiuDSId7vbxANj4YFVpbXkxNkFKm35s&output=z3mwepuDBMOnja63VsgAcCGwmmOPD-I5yFYOiyCvBcc&channel=FBDoImE2NP77XpNbdycjnwCkI0YjxLeqTKr8QGV9bAI&protocol=4CMm-Q0epGZ0SHQGeELoqqA2gfzFVCFwoiACYszyPM4&type=3pIgQ1TALsxZg7Zf0kZLkA4InWPfKONROAImuoufxKg (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9f1bffd310>, 'Connection to cdnswitch-mediahub.overon.es timed out. (connect timeout=15)'))
2024-12-11 10:05:15,617 - ERROR - URL Error http://cdnh4.iblups.com/hls/RMuwrdk7M9.m3u8: HTTPConnectionPool(host='cdnh4.iblups.com', port=80): Max retries exceeded with url: /hls/RMuwrdk7M9.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9f1adc1640>: Failed to resolve 'cdnh4.iblups.com' ([Errno -2] Name or service not known)"))
2024-12-11 10:05:15,767 - ERROR - URL Error https://cu.onliv3.com/livevd1/user2.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cu.onliv3.com', port=443): Max retries exceeded with url: /livevd1/user2.m3u8?PlaylistM3UCL (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))
2024-12-11 10:05:15,800 - ERROR - URL Error https://dysmuyxh5vstv.cloudfront.net/hls/atv2.m3u8: HTTPSConnectionPool(host='dysmuyxh5vstv.cloudfront.net', port=443): Max retries exceeded with url: /hls/atv2.m3u8 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f9f1adddc40>: Failed to resolve 'dysmuyxh5vstv.cloudfront.net' ([Errno -5] No address associated with hostname)"))
2024-12-11 10:05:27,206 - ERROR - URL Error http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=MJB6CUxvZcU: HTTPConnectionPool(host='free.fullspeed.tv', port=80): Max retries exceeded with url: /iptv-query?streaming-ip=https://www.youtube.com/watch?v=MJB6CUxvZcU (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9f1adc35c0>: Failed to resolve 'free.fullspeed.tv' ([Errno -2] Name or service not known)"))
